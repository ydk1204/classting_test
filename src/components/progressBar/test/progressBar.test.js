import React from 'react';
import renderer from 'react-test-renderer';
import ProgressBar from '../progressBar';

const value = 0;
const questionNumber = 3;

describe('ProgressBar', () => {
  it('renders', () => {
    const component = renderer.create(<ProgressBar value={value} questionNumber={questionNumber} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});