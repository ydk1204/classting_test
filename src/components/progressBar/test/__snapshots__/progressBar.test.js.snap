// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar renders 1`] = `
<div
  className="relative top-0 left-0 bg-slate-800 border-[1px] rounded-md w-full"
>
  <p
    className="w-full top-[-0.2em] absolute text-center text-white cursor-default"
  >
    0
  </p>
  <div
    className="bg-green-500 h-4 rounded-md transition-all duration-300 ease-in"
    style={
      Object {
        "width": "0%",
      }
    }
  />
</div>
`;
